src/styles/index.scss

```diff
@@ -1,3 +1,96 @@
+@use 'sass:map';
+
+$daybackground: #eee;
+$dayheight: 100px;
+$daywidth: 100px;
+$daycount: 31;
+$border-width: 1px;
+$days-of-week: (
+  mon: 0,
+  tue: 1,
+  wed: 2,
+  thu: 3,
+  fri: 4,
+  sat: 5,
+  sun: 6,
+);
+$boxheight: auto;
+
+@mixin start-day($start-day) {
+  $margin: map.get($days-of-week, $start-day);
+  .calendar--start-day-#{$start-day} .calendar__day:first-child {
+    margin-left: calc(#{$margin} * (#{$daywidth} + #{$border-width}));
+  }
+}
+
```

Comment:
In this task, you can do without a mixin, but if you use it, it must be in a separate file